### ISSUE TITLE
MWE for each level?

### ISSUE THREAD

**Issue by CarolinOdebrecht** [distantreading/GermanTextCollection#1]
Maybe it would help to provide a minimal working example for each encoding level when we have finished the schemes? Just a few text lines (not an entire novel) with all possbile elements which will show how the tei xml should "look like" in each level.

**Comment by lb42** [comment]
Yes, the ODD file should have examples for each element to replace the default TEI ones. And in different languages!
